src/main/resources/org/semux/api/swagger/v1.0.1.json

```diff
@@ -1,1361 +1,1573 @@
+                    }
+                ]
+            }
         },
-        "pendingTransactions" : {
-          "type" : "integer",
-          "format" : "int32"
-        }
-      }
-    },
-    "ListAccountsResponse" : {
-      "type" : "object",
-      "required" : [ "success" ],
-      "properties" : {
-        "success" : {
-          "type" : "boolean"
+        "/get_latest_block_number": {
+            "get": {
+                "summary": "Get latest block number",
+                "description": "Returns the number of the latest block.",
+                "operationId": "getLatestBlockNumber",
+                "produces": [
+                    "application/json"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/GetLatestBlockNumberResponse"
+                        }
+                    }
+                },
+                "security": [
+                    {
+                        "basicAuth": []
+                    }
+                ]
+            }
         },
-        "result" : {
-          "type" : "array",
-          "items" : {
-            "type" : "string"
-          }
+        "/get_peers": {
+            "get": {
+                "summary": "Get peers",
+                "description": "Returns connected peers.",
+                "operationId": "getPeers",
+                "produces": [
+                    "application/json"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/GetPeersResponse"
+                        }
+                    }
+                },
+                "security": [
+                    {
+                        "basicAuth": []
+                    }
+                ]
+            }
         },
-        "message" : {
-          "type" : "string"
-        }
-      }
-    },
-    "PeerType" : {
-      "type" : "object",
-      "properties" : {
-        "ip" : {
-          "type" : "string"
+        "/get_pending_transactions": {
+            "get": {
+                "summary": "Get pending transactions",
+                "description": "Returns all the pending transactions.",
+                "operationId": "getPendingTransactions",
+                "produces": [
+                    "application/json"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/GetPendingTransactionsResponse"
+                        }
+                    }
+                },
+                "security": [
+                    {
+      
```

Comment:
Convention would be this is first line of file for legibility.  I'm not sure the sorting of whole file is good.  Sorting of methods sections and response objects would be preferred.

You want name, auth, schema, description, at top